src/app/drive/services/usage.service.ts

```diff
@@ -10,9 +10,7 @@ export async function fetchUsage(): Promise<UsageResponse> {
     photosClient.photos.getUsage(),
   ]);
 
-  driveUsage.total += photosUsage;
-
-  return driveUsage;
+  return { ...driveUsage, photos: photosUsage };
```

Comment:
This way not all the space used will be shown in the main Drive view as the space used for photos is not added up.
How should it be displayed where it was already displayed?